lib/dialects/mssql/query.js

```diff
@@ -210,12 +211,12 @@ Query.prototype.formatError = function (err) {
     if (uniqueKey && !!uniqueKey.msg) {
       message = uniqueKey.msg;
     }
-    if (!!match[2]) {
```

Comment:
No idea, not sure if it's ever been correct, but it certainly isn't at the moment.